code/game/objects/items/explosives/mine.dm

```diff
@@ -286,3 +286,162 @@
 	customizable = TRUE
 	matter = list("metal" = 3750)
 	has_blast_wave_dampener = TRUE
+
+
+
+/obj/item/explosive/mine/bounding
+	name = "\improper M5 Bounding Mine"
+	desc = "The M5 Bounding Mine is a landmine that propels itself upwards when the tripwire is set off. It explodes mid-air and either maims, kills, or fucks up anything standing in the blast zone. The 'Bouncing Betty' has seen limited use since the year 2150 due to the IFF sensors on the earlier models were prone to failure, resulting in friendly casualties. Be careful to not trip this or you're going to be ending up in a hospital bed full of shrapnel."
+	icon = 'icons/obj/items/weapons/grenade.dmi'
+	icon_state = "m5"
+	force = 5.0
```

Comment:
A significant chunk of these variables are the same as the parent and don't need to be redefined.